src/Microsoft.Identity.Client/ApiConfig/Executors/PublicClientExecutor.cs

```diff
@@ -136,7 +136,10 @@ public PublicClientExecutor(IServiceBundle serviceBundle, PublicClientApplicatio
             coreUiParent.UseCorporateNetwork = _publicClientApplication.AppConfig.UseCorporateNetwork;
 #endif
 #endif
-            return ServiceBundle.PlatformProxy.GetWebUiFactory().CreateAuthenticationDialog(coreUiParent, requestContext);
+            return ServiceBundle.PlatformProxy.GetWebUiFactory().CreateAuthenticationDialog(
+                coreUiParent,
+                requestContext,
+                ServiceBundle.PlatformProxy);
```

Comment:
The new webUI needs to open the OS browser. This is actually platform AND os specific and even if the current code is only for .net core, I've decided to add the "openBrowser" functionality to platform proxy, to make testing easier.